docker/docker.go

```
@@ -508,6 +444,12 @@ func NewDockerPushStep(stepConfig *core.StepConfig, options *core.PipelineOption
 	}, nil
 }
 
+// Just to make life easier
```

Why are we ignoring error here?

---

I've removed the function and moved its code inline. The error is no longer ignored.